docs/docs/91-migrations.md

```diff
@@ -4,6 +4,8 @@ Some versions need some changes to the server configuration or the pipeline conf
 
 ## `next`
 
+- `WOODPECKER_DEFAULT_CLONE_IMAGE` got depricated use `WOODPECKER_DEFAULT_CLONE_PLUGIN`
+- Check trusted-clone-plugins by image name and tag (if tag is set)
```

Comment:
I'm not sure if we should add it to migrations, as normally an admin dont have to do anything even if he specified an image ...